container-action/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:24.04 AS builder
+
+LABEL Description="Custom RSK node image to execute Rootstock Integration Tests"
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    wget gnupg2 curl git ca-certificates mocha \
+    && apt clean
+
+# -- nodeJs ---------------------------------------------------------
+ENV NODE_VERSION  v20.14.0
+RUN mkdir -p /usr/local/nvm
+ENV NVM_DIR /usr/local/nvm
+
+RUN  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash \
+     && . $NVM_DIR/nvm.sh \
+     &&  nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/$NODE_VERSION/bin:$PATH
+
+# -- java ---------------------------------------------------------
+ENV JAVA_VERSION 17
+
+RUN apt-get  -y install "openjdk-$JAVA_VERSION-jdk"
+
+ENV JAVA_HOME="/usr/lib/jvm/java-$JAVA_VERSION-openjdk-amd64"
+
+RUN export JAVA_HOME="$JAVA_HOME"  \
+    && export PATH="$JAVA_HOME:$PATH"
```

Comment:
This is a no-op and adds an extra layer to the container. The reach of those `export` are for the whole `RUN` instruction, not for all the ones after. My understanding is that the previous `ENV JAVA_HOME=...` should be enough.